lib/net_mosq.c

```diff
@@ -635,6 +635,14 @@ int _mosquitto_socket_connect_step3(struct mosquitto *mosq, const char *host, ui
 		}
 		SSL_set_bio(mosq->ssl, bio, bio);
 
+    /*
+     * required for the SNI resolving
+     */
+    if(SSL_set_tlsext_host_name(mosq->ssl, host) != 1) {
+			COMPAT_CLOSE(mosq->sock);
+			return MOSQ_ERR_TLS;
+    }
+
```

Comment:
Looks like you have a different whitespace setting in your editor here.  you want to maintain the same code style as the surrounding code.

Reply:
ok, fixed